docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,92 @@ discussion: 5469
+
+In contrast, the data that is communicated via signals is not retained in the container, which makes it an appropriate channel for transmitting transient data. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol. Signals are not guaranteed to be ordered on delivery relative to other signals and ops, but they still provide a useful communication channel for impermanent and short-lived data.
+
+## Why are signals useful?
+Signals provide a communication channel for sharing short-lived information that does not need to be persisted in the Fluid op stream.
+
+By sending signals, you avoid the storage and sequencing of data that will not be relevant or useful in the long-term.
+
+Signals are the most appropriate data channel in many user presence scenarios, where each user has the responsibility of sharing their current presence state to other connected users. In these scenarios, current presence data is short-lived, past presence state is irrelevant, and the shared data is not persisted on disconnect.
+
+## How can I use signals in Fluid?
+The `Signaler` DataObject can be used to send communications via signals in a Fluid application. `Signaler` allows clients to send signals to other connected clients and add/remove listeners for specified signal types.
+
+# Signaler
+
+## Creation
+Just like with DDSes, you can include `Signaler` as a shared object you would like to load in your `FluidContainer` schema.
+
+Here is a look at how you would go about loading `Signaler` as part of the initial objects of the container:
+
+```typescript
+const containerSchema: ContainerSchema = {
+    initialObjects: {
+        signaler: Signaler,
+    },
+};
+
+const { container, services } = await client.createContainer(containerSchema);
+
+const signaler = container.initialObjects.signaler as Signaler;
+```
+
+`signaler` can then be directly used in your Fluid application!
+
+For more information on using `ContainerSchema` to create objects please see [Data modeling](https://fluidframework.com/docs/build/data-modeling/).
+
+## API
+`Signaler` provides a few simple methods to send signals and add/remove listeners to specific signals as well:
+- `submitSignal(signalName: string, payload?: Jsonable)` - Sends a signal with a payload to its connected listeners
+- `onSignal(signalName: string, listener: SignalListener)` - Adds a listener for the specified signal. Similar behavior as EventEmitter's `on` method.
+- `offSignal(signalName: string, listener: SignalListener)` - Removes a listener for the specified signal. Similar behavior as EventEmitter's `off` method.
```

Comment:
These are puzzling without at least a one-sentence description of what the data type SignalListener is. 

Also, is the `listener` parameter a handler? I suspect it is, in which case it should be called `handler`. A listener and a handler are not quite the same thing.

Reply:
I think the usage of the term listener stems from the [EventEmitter Node.js class](https://nodejs.org/api/events.html#emitteroneventname-listener) and it's `emitter.on(eventName, listener)` and `emitter.off(eventName, listener)` methods. In the documentation, they define listener as "an event handler function" or a "callback function", so I thought the usage be appropriate.

Reply:
OK. If this term has become industry-standard, we can use it.